pkg/api/v1/types.go

```diff
@@ -710,6 +710,8 @@ type ISCSIVolumeSource struct {
 	IQN string `json:"iqn"`
 	// iSCSI target lun number.
 	Lun int `json:"lun"`
+	// Optional: Defaults to 'default' (tcp). iSCSI interface name that uses an iSCSI transport.
```

Comment:
This comment is not sufficient for me.  What is an interface here?  Can you be clearer on what this is doing?